src/SFA.DAS.ApprenticeCOmmitments.Migrations/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.EntityFrameworkCore;
+using SFA.DAS.ApprenticeCommitments.Data.Models;
+
+namespace SFA.DAS.ApprenticeCommitments.Migrations
+{
+    // From das-apprentice-commitments-api\src\SFA.DAS.ApprenticeCommitments.Migrations
+    // > dotnet ef dbcontext script
+
+    public class DbContext : ApprenticeCommitmentsDbContext
+    {
+        public DbContext() : base(
+            new DbContextOptionsBuilder<ApprenticeCommitmentsDbContext>()
+                .UseSqlServer("Data Source=.;Initial Catalog=SFA.DAS.ApprenticeCommitments.Migrated;Integrated Security=True;Pooling=False;Connect Timeout=30")
+                .Options)
+        {
+        }
+    }
+}
```

Comment:
Migrations was probably a bad name for it.  This project is here so that we can inspect the SQL that EF would generate if it were in charge of the db schema, and make sure that we don't have any glaring mismatches between EF and our manual SQL.

I'll rename it to hopefully make that clearer.